.circleci/config.yml

```diff
@@ -83,11 +83,23 @@ jobs:
                 else
                     touch test_preparation/test_list.txt
                 fi
+            - run: python utils/tests_fetcher.py --filters tests examples | tee examples_tests_fetched_summary.txt
+            - store_artifacts:
+                path: ~/transformers/examples_tests_fetched_summary.txt
+            - run: |
+                if [ -f test_list.txt ]; then
+                    cat test_list.txt
```

Comment:
Just tiny question: do we also want to `cat` or `echo no file named ...` the file `test_list.txt` in the above block too?

Reply:
Oh those were debug statements, I'll remove them. The artifact uploaded should be enough for most cases!